wdl-engine/src/eval/v1/task.rs

```diff
@@ -213,262 +251,175 @@ impl<'a> TaskEvaluator<'a> {
             )
         })?;
 
-        let mut execution = self.engine.backend().create_execution(root)?;
-        match document.node().ast() {
-            Ast::V1(ast) => {
-                // Find the task in the AST
-                let definition = ast
-                    .tasks()
-                    .find(|t| t.name().as_str() == task.name())
-                    .expect("task should exist in the AST");
+        let ast = match document.node().ast() {
+            Ast::V1(ast) => ast,
+            _ => return Err(anyhow!("document is not a 1.x document").into()),
```

Comment:
Does this have the potential to be shown to a user? If so, I wonder if an error message like `task evaluation is not supported for documents that aren't WDL v1.x` (or similar).

Reply:
I can update this and the workflow one.